lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
That is a good argument (if you'll pardon the pun). We definitely can't break the whole cli, so it'll at least have to be camel-case.

I do quite like `buildArg` over `arg` alone though - it feels clearer, and if you need it to be terse you can use the alias. And the `build` isn't always superfluous anyway, in `resin deploy --arg`, it's not clear whether the arg is for the build part or the deployed code. `resin deploy --buildArg` is very clear.